### PR TITLE
Add condor devcontainer features

### PR DESCRIPTION
**Pull request by trestrantham**

### CODE REVIEW

File: _data/collection-index.yml [hunk 1]
```diff
@@ -292,4 +292,9 @@
   maintainer: Hamish Cox
   contact: https://github.com/HamishWHC/cs3231-devcontainer/issues
   repository: https://github.com/HamishWHC/cs3231-devcontainer
-  ociReference: ghcr.io/hamishwhc/cs3231-devcontainer
\ No newline at end of file
+  ociReference: ghcr.io/hamishwhc/cs3231-devcontainer
+- name: Condor Container Features
+  maintainer: @trestrantham
```
Comment (samruddhikhandale) [inline _data/collection-index.yml:297]: The yml parser does not allow `@` as the first character of the token, hence it results into build failures. Can you either remove `@` or double quote the value? Thanks!

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
Thanks, left one comment!

File: _data/collection-index.yml (hunk 1 above)
Comment (samruddhikhandale) [reply _data/collection-index.yml:297]: @trestrantham Pinging on this thread as a friendly reminder! :)

Comment (trestrantham) [reply _data/collection-index.yml:297]: @samruddhikhandale Thanks for the nudge! I actually ended up going a different direction with this (using local features) while I work out a different approach here. I will close this for now and open a new PR if/when it's ready for consumption. Cheers!

**Pull request closed without merging**